style.css

```diff
@@ -65,7 +65,8 @@ html *[lang] { display: none; }
 html[lang=en] *[lang=en],
 html[lang=es] *[lang=es],
 html[lang=fr] *[lang=fr],
-html[lang=ja] *[lang=ja] {
+html[lang=ja] *[lang=ja]
```

Comment:
missed a comma :)

Reply:
Wait, did I fix this *while* you were reviewing? 😄

Reply:
Yeah I put a comment and there was a fix commit in like 15 secs. And I thought well that was fast :D